pkg/worker/v1/rpc/worker.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+
+// Versioning via package name, for future-proofing
+package wharf.worker.v1;
+
+option go_package = "github.com/iver-wharf/wharf-cmd/pkg/proto/worker/v1";
```

Comment:
Conventionally, and personally preferrably, gRPC definitions should live in the `api` dir instead of the `pkg`.

And the `proto` part in the path is excessive IMO.

Suggest moving it to `github.com/iver-wharf/wharf-cmd/api/workerapi/v1`

And then omit the `v1` part from the Go package, meaning having the Go wrapper in `pkg/workerapi`